R/symptome_compartment_class.R

```diff
@@ -0,0 +1,226 @@
+#' An SEIR model for compartments of different symptoms
+#'
+#' Note:
+#' 1) Sum of poulation equals to 1 at any time point.
+#' An error will be reported the sum of initial population is not equal to 1.
```

Comment:
I'm not sure this needs to be here as it'll be handled in the code. Also, just about maintainability: if we removed that error check, we'd also have to remove this.